azure-pipelines.yml

```diff
@@ -30,7 +30,7 @@ jobs:
            curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
            python3.6 get-pip.py && \
            pip3 --version && \
-           pip3 install setuptools wheel numpy cython==0.29.14 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath matplotlib --user && \
+           pip3 install setuptools wheel numpy cython==0.29.14 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath matplotlib==3.1.3 --user && \
```

Comment:
If we really want the latest stable version of matplotlib  we could use a temporary workaround suggested upstream: `LANG=C.utf-8 pip install matplotlib`

Feel free to push this change into my PR if you want. CI is all green and it looks like they will remove the non-ascii characters from their README in 3.2.1 so we could just unpin then for 32-bit. And the broader issue will disappear with Python 3.7.